### ISSUE TITLE
4.3.0 detected as malware

### ISSUE THREAD

**Issue by dominic-03** [axstin/rbxfpsunlocker#1819]
After downloading Roblox FPS Unlocker 32 bit version 4.3.0, I extracted it. Immediately after, my antivirus removed it from my system from containing a virus. While I don't believe the program is actually a virus, there is something in it [triggering multiple antivirus engines](https://www.virustotal.com/gui/file/a27f747390816539214b01ab89b998ada272d51a42b89cb1a448122815bb981a/detection). 4.2.0 has no problems and I continue to use it.

**Comment by axstin** [comment]
Yes, the 32-bit versions of RFU have heavy false positives for reasons unknown to me (this is the case with v4.2.0 as well). If you have a 64-bit OS, as the majority of people do, please use the 64-bit version of RFU.
